tsl/test/sql/dist_compression.sql

```diff
@@ -69,23 +69,44 @@ FROM show_chunks('compressed') AS chunk
 ORDER BY chunk
 LIMIT 1;
 
-
 -- Check that one chunk, and its replica, is compressed
 SELECT * from chunk_compression_stats( 'compressed')
 ORDER BY chunk_name, node_name;
 select * from hypertable_compression_stats('compressed'); 
 
--- Compress twice to generate NOTICE that the chunk is already compressed
-SELECT compress_chunk(chunk, if_not_compressed => true)
+--- Decompress the chunk and replica
+SELECT decompress_chunk(chunk)
 FROM show_chunks('compressed') AS chunk
 ORDER BY chunk
 LIMIT 1;
 
--- Decompress the chunk and replica
+-- Compress odd numbered chunks
+SELECT compress_chunk(chunk) FROM
+(
+  SELECT *, row_number() OVER () AS rownum
+  FROM show_chunks('compressed') AS chunk
+  ORDER BY chunk
+) AS t
+WHERE rownum % 2 = 1;
+
+-- Compress twice to notice idempotent operation
```

Comment:
Good practice to check the test as well: suggest to add a check that the odd chunks really are compressed and that the statistics show the correct status.